README.md

```diff
@@ -35,6 +36,12 @@ $ python3 setup.py install
 
 ## Examples
 
+### Crates
+
+```
+$ perceval crates --category
```

Comment:
You have to set the category. It doesn't work that way.

Reply:
Should I specify the options here (eg [summary|crates])? Or should I removed that option and just leave 'perceval crates'?

Thanks!

Reply:
Just write an example with any of the categories available